### ISSUE TITLE
Move components to partners site

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#3240]
We should move all of these components out of UIC and into the partners site. All of these components are currently only used on the partners site, and a number of them include either/both a lot of layout or Bloom-specific business logic.

The below components are used only once.
`FormSignInAddPhone`
`FormSignInMFACode`
`FormSignInMFAType`

The below components are used more than once.
`FormTerms`
`NavigationHeader`
`StatusAside`
`StatusBar`

We should move all of them to a components/ folder in the partners site that will house components that we use just once, and ones that we reuse.

I would suggest the following folder structure all nested under a src folder (so that all site content is in once place - next should already support the pages directory living within src), where the components folder is organized by page structure for components used only once (the partners site is already mostly structured this way).

pages/
--application/
--listings/
--settings/
----index.tsx

components/
--application/
--listings/
--settings/
----ExampleSettingsComponent.tsx
----ExampleSettingsComponent.scss
----ExampleSettingsComponent.test.tsx
--shared/ (components used in more than one file)
----NavigationHeader.tsx
----NavigationHeader.scss
----NavigationHeader.test.tsx

AC:
Partners site has the new directory structure. Will require moving everything under src, organizing the existing components that live in the src directory into the new components directory, copying and pasting the listed components out of UIC into these folders, and importing these versions instead of the UIC versions where they are used.

Near future work separated into different ticket:
Run unit tests in the components folder

**Comment by emilyjablonski** [comment]
QA Notes: There are no code changes here, just files moved around!
